lalrpop/src/build/mod.rs

```diff
@@ -348,8 +348,14 @@ fn emit_recursive_ascent(session: &Session, grammar: &r::Grammar) -> io::Result<
                                                         "super",
                                                         &mut rust)),
 
-            r::LrCodeGeneration::TestAll =>
-                try!(lr1::codegen::test_all::compile(&grammar, user_nt, start_nt, &states, &mut rust)),
+            r::LrCodeGeneration::TestAll => {
+                try!(lr1::codegen::test_all::compile(&grammar, user_nt, start_nt, &states, &mut rust));
+                rust!(rust,
```

Comment:
wait, what is happening here? I never did figure out why this `pub use` is needed...